### DISCUSSION TITLE
Alterations to dotenv.txt have no effect

### DISCUSSION THREAD

**Discussion by BouchardJ** [jef/streetmerchant#2327] [Help]
I saved a copy as dotenv.txt and changing the values within it is not changing the behavior of the program. For example, setting SHOW_ONLY_SERIES=3090 does not get rid of all the other series; the program continues to monitor all the default web pages. I read all the documentation and can't figure out what I've done wrong.

**Comment by ReflexivR** [comment]
Try dotenv with no .txt extension
